### ISSUE TITLE
Control theory and prediction

### ISSUE THREAD

**Issue by aceeccc** [openconnectome/Statistical-Connectomics-Class#188]
Control theory can be used for correcting a system towards what the programmer/scientist wants the output to be. How feasable is it to find the state space model of a brain and control the brain activity of a person? or for predicting the future patterns of a brain?

**Comment by dlee138** [comment]
How would one find the state space model of a brain in the first place?  If it is only a "model" then i believe it will be quite limited for predicting/controlling the brain, simply because of the raw number of input, output and state variables that are possible.  I feel like a truly accurate state space model isn't feasible at this point and an extremely simple one won't really predict anything too useful.

**Comment by aceeccc** [comment]
I found a paper that uses a method called "sparse regression" to deal with the size of the data: http://www.sciencedirect.com.proxy1.library.jhu.edu/science/article/pii/S1053811911002515

**Comment by ajulian3** [comment]
I don't believe this theory will be beneficial or plausible in long term application. Specifically, the neural plasticity within the brain along with limited information will make applying control theory quite difficult. What are further applications of sparse regression in connectomics research?

**Comment by akim1** [comment]
I guess it probably depends on the level at which you are talking about. But I think at the neuronal level, brain activities are nonlinear in nature. Depending on the condition, they're also probably highly sensitive to initial condition (i.e., chaos theory). Which means that unless you have perfect measurement resolution, you probably won't be able to predict the future of a brain activity.

**Comment by mrjiaruiwang** [comment]
That would be pretty hard. You would need to make some assumptions to reduce the problem into an LTI system in order for us to control it. If you leave it to be nonlinear, it would be very difficult to design a controller that stays within the stable bounds, especially in a system where it is difficult to measure.

**Comment by mblohr** [comment]
Although this is specifically for epilepsy models, one approach to modelling brain state spaces is provided in ( http://www.pnas.org/content/111/49/E5321.abstract ) with extension to control theory in ( http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4347430/ ).
